sqlmesh/core/snapshot/definition.py

```diff
@@ -756,7 +756,7 @@ def version_get_or_generate(self) -> str:
     def physical_schema(self) -> str:
         if self.physical_schema_ is not None:
             return self.physical_schema_
-        _, schema, _ = parse_model_name(self.name)
+        schema = parse_model_name(self.name)[1]
```

Comment:
i wonder if we just do to_table.schema

Reply:
Good idea going to just replace this with `to_table`

Reply:
https://github.com/TobikoData/sqlmesh/pull/1130/commits/79db74c91162a57081296be3f2f2707bf90b4fb2